<!-- BEGIN KEY -->
public class javax.activity.ActivityCompletedException extends java.rmi.RemoteException
<!-- END KEY -->
<!-- BEGIN DATA -->
 This exception may be thrown on any method for which Activity context is
 accessed and indicates that ongoing work within the Activity is not
 possible. This may be because the Activity has been instructed to complete
 with CompletionStatusFailOnly or has ended as a result of a timeout. This 
 exception will be propagated across ORB boundaries via an
 org.omg.CORBA.ACTIVITY_COMPLETED system exception. An application should
 handle this error by attempting to complete the Activity.

<!-- END DATA -->
<!-- BEGIN KEY -->
public javax.activity.ActivityCompletedException()
<!-- END KEY -->
<!-- BEGIN DATA -->
 Constructs a new instance with null as its detail message.

<!-- END DATA -->
<!-- BEGIN KEY -->
public javax.activity.ActivityCompletedException(String message)
<!-- END KEY -->
<!-- BEGIN DATA -->
 Constructs a new instance with the specified detail message.

 @param message the detail message.

<!-- END DATA -->
<!-- BEGIN KEY -->
public javax.activity.ActivityCompletedException(Throwable cause)
<!-- END KEY -->
<!-- BEGIN DATA -->
 Constructs a new throwable with the specified cause.

 @param cause a chained exception of type
 <code>Throwable</code>.

<!-- END DATA -->
<!-- BEGIN KEY -->
public javax.activity.ActivityCompletedException(String message, Throwable cause)
<!-- END KEY -->
<!-- BEGIN DATA -->
 Constructs a new throwable with the specified detail message and cause.

 @param message the detail message.

 @param cause a chained exception of type
 <code>Throwable</code>.

<!-- END DATA -->
